src/components/collection-card/components/company-logo/types.ts

```diff
@@ -1,12 +1,6 @@
-export enum CompanyLogoOption {
-  docker = 'docker',
-  github = 'github',
-  'microsoft-azure' = 'microsoft-azure',
-  oci = 'oci',
-  'google-cloud' = 'google-cloud',
-  'terraform-cloud' = 'terraform-cloud',
-  aws = 'aws',
-}
+import { CompanyLogoOption } from 'lib/learn-client/types'
```

Comment:
I've moved `CompanyLogoOption` into `learn-client/types`. This felt more natural when constructing the `ProductPage` block types, and it felt intuitive to think of our component as filling out its logo options based on possibilities defined by `learn-client`, rather than the other way around.

Definitely happy to adjust here though, I can see it both ways 🤔

Reply:
I agree, it makes sense there 👌